db/migrate/20200505201706_create_demux_transmissions.rb

```diff
@@ -10,6 +10,7 @@ def change
       t.string :response_code
       t.jsonb :response_headers
       t.text :response_body
+      t.jsonb :context
```

Comment:
🎉 to jsonb. We could always add db specific migrations/adapters later on if support is desired